src/WebJobs.Extensions.DurableTask/EtwEventSource.cs

```diff
@@ -205,6 +205,26 @@ private EtwEventSource()
             this.WriteEvent(211, TaskHub, AppName, SlotName, FunctionName, FunctionState, Version ?? "", InstanceId, Details, StatusCode, Reason, FunctionType, ExtensionVersion, IsReplay, LatencyMs);
         }
 
+        [Event(212, Level = EventLevel.Error)]
+        public void EventGridNotificationException(
```

Comment:
This seems very similar to `EventGridNotificationFailed` (Event ID 211). Do we really need a new event or can we use the existing event?